modules/tensor_mechanics/doc/content/documentation/systems/Modules/TensorMechanics/LineElementMaster/index.md

```diff
@@ -1 +1,73 @@
-!include /Modules/TensorMechanics/LineElementMaster/LineElementAction.md
+# Line Element Action System
+
+!syntax description /Modules/TensorMechanics/LineElementMaster/LineElementAction
+
+## Description
+
+The LineElement Action is a convenience object that simplifies part of the
+mechanics/dynamics system setup for beam or truss elements.
+
+## Truss Elements: Constructed MooseObjects
+
+The `LineElement` Action can be used to construct the kernels, strain materials, and displacement variables for a simulation using +Truss Elements+.
+
+!alert note title=Set the Truss Parameter
+A truss element is chosen by setting +`truss = true`+ in the input block.
+
+!table id=lineElement_Truss_action_table caption=Correspondence Among Action Functionality and MooseObjects for the `LineElement` Action used in a +Truss Elements+ Simulation
+| Functionality     | Replaced Classes   | Associated Parameters   |
+|-------------------|--------------------|-------------------------|
+| Create truss elements | (*none*) | `truss = true` |
+| Add the displacement variables | [Variables](systems/Variables/index.md) | `add_variables`: boolean
+| Calculation of stress divergence for a +Truss+ element | [StressDivergenceTensorsTruss](/StressDivergenceTensorsTruss.md) | `displacements` : a string of the displacement variables |
+
+This action can also optionally set up the AuxVariables and AuxKernels combinations for post processing outputs. See the documentation for the [TensorMechanics Master action](../Master) for the generation of output AuxVariables and AuxKernels.
+
```

Comment:
The Line element action does not set up aux variables and aux kernels for post-processing output yet. Also, there is no stress/strain tensor for beam and truss elements. So, I don't think the options in Tensor Mechanics Master Action can be used to set up the required object for post processing.

Reply:
Thank you for catching this mistake @sveerara! I'll get it fixed in my next push